proposals/POEM-1-proposal-for-openwhisk-enhancements.md

```diff
@@ -0,0 +1,65 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Title
+Process for introducing an OpenWhisk Enhancement (POEM)
+
+## Status
+* Current state: Draft
+* Author: @style95
+
+## Summary
+
+Introduce a process for suggesting and implementing a substantive OpenWhisk Enhancement (POEM).
+A developer or group of developers working together to propose and implement a major new feature or functionality, a new subsystem, or a breaking change should follow the process described herein and open a proposal for consideration under this directory.
+
+## Motivation
+
+As the project grows, more and more issues are getting complex and require multiple parties and an extended period of time to develop them.
+We can incubate, manage, and elaborate new ideas in OpenWhisk with a standard way and a well-tracked artifact.
+It would enhance the discoverability of proposals and help community members to involve in the project better.
+
+## Proposed changes
+
+### Procedures
+1. Create a pull request to describe your proposal with [this template](./POEM-N-template.md). The initial state of a proposal should be _"Draft"_.
+2. [Create a corresponding issue]((/issues/new?template=proposal.md)) to propose a new change based on [this template](../github/ISSUE_TEMPLATE/proposal.md). It is mainly used to track discussion history.
+3. Discuss the proposals with any channels(issue comments, mailing list, Slack, etc) to form a consensus and update your proposal based on comments as much as needed.
+4. When members form a rough consensus for the proposal. The proposal owner can request a vote via the dev mailing list.
+5. The voting process follows the [Apache Voting guideline](https://www.apache.org/foundation/voting.html). The PR can be merged with the _"In-progress"_ state if the voting is successfully closed without any veto.
```

Comment:
Votes may pass by silent assent. Do we want to allow silent assent or require at least 3 +1s to show buy in?
cc @dgrove-oss for input.

Reply:
Not sure how best to formalize this, but I'd hope there would be at least some discussion and positive comments.  The pattern to avoid is a proposal being under-discussed and then merged by silent consent.